src/components/Comments/Comment.js

```diff
@@ -0,0 +1,96 @@
+import {
```

Comment:
please review this docs, it might be helpful https://mui.com/material-ui/guides/minimizing-bundle-size/

Reply:
thanks! I didn't know. I can change those imports